client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import handlers from './wpcom';
+
+export const middleware = store => next => action =>
+	has( handlers, action.type )
+		? handlers[ action.type ].forEach( handler => handler( store, action ) )
```

Comment:
Do we need to pass the entire `store` object as the first argument to the handler, or would `dispatch` alone be sufficient? I guess I'm wondering: do we want to deter developers from accessing state via `getState` ?